src/oc_erchef/apps/oc_chef_wm/src/chef_wm_stats.erl

```diff
@@ -0,0 +1,59 @@
+%% -*- erlang-indent-level: 4;indent-tabs-mode: nil; fill-column: 92 -*-
+%% ex: ts=4 sw=4 et
+%%% @doc
+%%% REST resource for reporting chef server stats
+%%% @end
+%% Copyright 2017 Chef Software, Inc. All Rights Reserved.
+%%
+%% This file is provided to you under the Apache License,
+%% Version 2.0 (the "License"); you may not use this file
+%% except in compliance with the License.  You may obtain
+%% a copy of the License at
+%%
+%%   http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing,
+%% software distributed under the License is distributed on an
+%% "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+%% KIND, either express or implied.  See the License for the
+%% specific language governing permissions and limitations
+%% under the License.
+%%
+
+-module(chef_wm_stats).
+
+-ifdef(TEST).
+-compile(export_all).
```

Comment:
[nit] This doesn't export more than `-export()` already exports, does it?

Reply:
It should export everything in the file (for any tests to use them).